linkml/generators/plantumlgen.py

```diff
@@ -50,6 +51,11 @@ class PlantumlGenerator(Generator):
     directory: Optional[str] = None
     kroki_server: Optional[str] = "https://kroki.io"
     load_image: bool = True
+    tooltips_flag: bool = False
+
+    def __post_init__(self):
+        self.schemaview = SchemaView(self.schema)
+        super().__post_init__()
```

Comment:
Thank you for the contribution @jsnyder-csdisco!! 

`plantumlgen.py` unfortunately is one of the generators in LinkML's suite of generators that follows the _old_ visitor design pattern as opposed to the _new_ SchemaView based approach that we are using to implement generators.

You're brining in SchemaView and using it in code, which is great, that's the direction we want to be heading towards, but I'm wondering if it would be best to include your changes after an entire overhaul/transition to SchemaView for `plantumlgen.py`.

How important/urgent is your fix?

Reply:
We can skip the tooltips part but the mixins just break plantuml and there are other bugs lurking in there as well because of the spacing bugs.

Reply:
The tool tips depend on the schema view to get the description. If there is another way to get descriptions without schema views I am happy to adjust it.

Reply:
Yup, you should be able to get the description for the LinkML object, which in this case is 'class' by simply getting the value associated with the description property (using the dot syntax).

In your case it would be:

```python
cls.description
```